spec/form_models/publishers/job_application/tag_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+module Publishers
+  module JobApplication
+    RSpec.describe TagForm, type: :model do
+      subject(:tag_form) { described_class.new(job_applications:, status:, origin:, validate_status:) }
+
+      let(:job_applications) { create_list(:job_application, 2, :status_submitted) }
+      let(:status) { "shortlisted" }
+      let(:origin) { "submitted" }
+
+      describe "validation" do
+        context "when validate status true" do
+          let(:validate_status) { true }
+
+          it { is_expected.to validate_length_of(:job_applications) }
+          it { is_expected.to validate_presence_of(:status) }
+          it { is_expected.to validate_inclusion_of(:status).in_array(%w[submitted unsuccessful reviewed shortlisted interviewing]) }
+        end
+
+        context "when validate status false" do
```

Comment:
Would it make sense to test that the status presence/inclusion is not validated in this case?

Reply:
when the value of  ``validate_status``   evaluates to  truthy,  the validation occurs. 
I can rename the contexts to be more explicit.

Reply:
Yes got that bit, but as I see the both contexts, the tests are not covering that the presence & inclusion of the `status` is **only** validated when the `validate_status` is truth.

The `status` validation could also be happening when `validate_status: false` and the current tests would still pass. 

Does it make sense? Not sure I'm explaining myself very well.